src/pages/ViewPdf/index.tsx

```diff
@@ -1,5 +1,16 @@
+import PageWrapper from "./styles"
+import GlobalStyle from "../../styles/styles"
+import HomeButton from "../../components/HomeButton"
+
 const ViewPdf = () => {
-  return <div>View PDF</div>
+  return (
+    <PageWrapper>
+      <HomeButton />
```

Comment:
I think you can use the button by itself here. No need to create a component for this